CHANGELOG.md

```diff
@@ -117,7 +121,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Initial release
 
-[Unreleased]: https://github.com/GoogleCloudPlatform/functions-framework-python/compare/v2.3.0...HEAD
+[Unreleased]: https://github.com/GoogleCloudPlatform/functions-framework-python/compare/v2.4.0-beta.1...HEAD
+
```

Comment:
nit: Sorry I didn't catch this before, I don't think we should newly add a newline here unlike before.